client/views/join_channel.tpl

```diff
@@ -0,0 +1,6 @@
+<form id="join_channel_{{id}}" class="join_form" method="post" action="" style="display: none;">
+	<input type="text" class="input" name="channel" placeholder="#channel" pattern="[^\s]+" maxlength="200" title="Should be a valid channel name">
+	<input type="password" class="input" name="key" placeholder="channel key" pattern="[^\s]+" title="Should be a valid channel key" maxlength="200" >
```

Comment:
Can you add "(optional)" in the placeholder? New people might be wondering what a channel key is when they want to join.
Also I would just do "Name" (instead of "#channel") and "Key (optional)".

Reply:
Did you miss this comment? `Name`/`Key (optional)`